cmake/modules/protobuf.cmake

```diff
@@ -19,6 +19,7 @@ else()
 	set(PROTOC "${PROTOBUF_SRC}/target/bin/protoc")
 	set(PROTOBUF_INCLUDE "${PROTOBUF_SRC}/target/include")
 	set(PROTOBUF_LIB "${PROTOBUF_SRC}/target/lib/libprotobuf.a")
+	set(PROTOC_LIB "${PROTOBUF_SRC}/target/lib/libprotoc.a")
```

Comment:
Why do we need to track this file?

Reply:
Likely a leftover, I will clean up eventually.